images/img-builder.sh

```diff
@@ -25,6 +26,7 @@ repositories:
 EOF
 
 # Create root-tree for COS_RECOVERY
+luet install --system-target efi -y system/grub2-efi-image
```

Comment:
I'd really like to see at this point those packages being sourced in the `manifest.yaml` file in a specific field. So it's really easy to switch off default recovery packages, grub artifacts and configs in case of need

Reply:
Well I am not sure if it will make the manifest a bit confusing, since these will be four new sub sections of packages. So I'd list them in manifest with something like:

```yaml
packages:                                                     
  rootfs:                                                                       
  - system/cos                                                                  
  uefi:                                                                         
  - live/grub2-efi-image                                                        
  isoimage:                                                                     
  - live/grub2                                                                  
  - live/grub2-efi-image                                                        
  - recovery/cos-img 

disk_packages:
  efi:
  - system/grub2
  grub2:
  - system/grub2-artifacts
  system-img:
  - recovery/cos-img
```

All in different sections since they all have a different `--system-target`.  Or do you have something different in mind?

Reply:
I see, yep indeed `packages` should be `iso_packages` at this point to avoid any confusion. But I think for the time being it's fine also if we just annotate in comments along the lines to explain what are those sections about.

Yup that would make sense, but I would not block this PR from being merged at this point as it looks outside the scope of this PR and can be iterated over in a second round of changes :+1:

Reply:
Agree, I just created a follow up issue so we do not forget.